editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditFeaturesWithFeatureLinkedAnnotationSample.java

```diff
@@ -225,10 +226,17 @@ else if (selectedFeature.getGeometry().getGeometryType() == GeometryType.POINT)
       EditAttributesDialog editAttributesDialog = new EditAttributesDialog(selectedFeature);
 
       // show the dialog and wait for the user response
-      editAttributesDialog.showAndWait();
+      Optional<Boolean> updatingFeature = editAttributesDialog.showAndWait();
 
-      // update the selected feature's feature table
-      updateAttributes(selectedFeature);
+      // if the user chose to update the feature's attributes
+      if (updatingFeature.isPresent()) {
+        // update the selected feature's feature table
+        updateAttributes(selectedFeature);
+      }
+      else{
```

Comment:
extra space needed between else and bracket 
```suggestion
      else {
```